orttraining/orttraining/core/framework/gradient_graph_builder.cc

```diff
@@ -123,8 +123,9 @@ NodeSet GradientGraphBuilder::BFSWithStopGradient(const std::unordered_set<std::
     std::vector<const Node*> nodes = graph_->GetConsumerNodes(name);
     for (const Node* node : nodes) {
       int input_index = graph_utils::GetNodeInputIndexFromInputName(*node, name);
-      const std::unordered_set<size_t>* edges = GetStopGradientEdges(*node);
-      if (edges != nullptr && edges->count(input_index)) {
+      std::unordered_set<size_t> edges;
```

Comment:
I don't understand the reason behind this change. Can you please explain?

Reply:
Ok, let me add more details in the PR description. :)

Reply:
removed gradient build stuff, only fix a constant creation bug. Will create a new one if gradient builder still need the fix.